TestModels/SimpleTypes/SimpleEnum/src/Index.dfy

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 include "SimpleEnumImpl.dfy"
 
-module {:extern "simpletypessmithyenuminternaldafny" } SimpleEnum refines AbstractSimpleTypesSmithyEnumService {
+module {:extern "simple.types.smithyenum.internaldafny" } SimpleEnum refines AbstractSimpleTypesSmithyEnumService {
```

Comment:
Drop the `.`

Reply:
We drop `.` using a script but we don't push the dropped `.` file as of now. We can reevaluate this if needed.

Reply:
I see some conversation regarding this here: https://github.com/smithy-lang/smithy-dafny/pull/511

Reply:
Ahh so we are still using the script. I had deprecated the script because I had started to push this on a branch.